DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -845,8 +854,20 @@ if($sparkCreation -eq 'y') {
 
     $tokens = Get-Tokens
 	if ($useDatabricks -eq 'n') {
+
+        $sparkTemplate = "Spark-Template.json"
+        $sparkParameter = "Spark-parameter.json"
+
+        $version = ($HDInsightVersion -split '\.')[0]
```

Comment:
Why is the split needed ?

Reply:
The HDInsight cluster version could be '3.6' or '4.0'. Doing a split with '.' here is to extract the major version number (like 3 from '3.6', 4 from '4.0'. ). And then, $version would be converted to INT and compare with the version number. The current assumption is that HDInsight version 4 and above would support the auto-scaling feature.

Reply:
looks the auto-scale feature is available for HDInsight3.6 as well. Removed this version check